README.rst

```diff
@@ -107,11 +256,6 @@ Configuration
 Operations
 ==========
 
-Bootstrapping Consumers
------------------------
-
-.. todo:: Rewrite this as part of the tutorial using pgbench after the replication worker is done.
-
```

Comment:
This section was talking about what is now called "replication" so it wasn't relevant.  I do want to do a pgbench example, but I had an idea today to do that with a Docker image so I want to try that first.